src/renderers/common/Textures.js

```diff
@@ -445,10 +443,16 @@ class Textures extends DataMap {
 	 */
 	_destroyTexture( texture ) {
 
-		this.backend.destroySampler( texture );
-		this.backend.destroyTexture( texture );
+		if ( this.has( texture ) === true ) {
```

Comment:
This is to make sure the following does not break the dispose logic:
```js
renderTarget.dispose();
renderTarget.texture.dispose();
```
Since calling `dispose()` on the render target disposes all of its textures, it's important to ensure `dispose()` called on texture does not trigger redundant operations.